extensions/locals.tf

```diff
@@ -1,39 +1,83 @@
 locals {
-  plans = yamldecode(file("${path.module}/plans.yml"))
+  plans = {
+    opensearch        = yamldecode(file("${path.module}/../opensearch/plans.yml"))
+    postgres          = yamldecode(file("${path.module}/../postgres/plans.yml"))
+    elasticache-redis = yamldecode(file("${path.module}/../elasticache-redis/plans.yml"))
+  }
 
   # So we don't hit a Parameter Store limit, filter environment config for extensions so it only includes the defaults (`"*"`) and the current environment
   extensions_for_environment = {
-    for k, v in var.args.services :
-    k => merge(v, {
```

Comment:
I found myself having to think to much about what all the `k`s and `v`s actually were, so this includes some more meaningful names to improve the grokkability of this code.